examples/plaintext.rs

```diff
@@ -0,0 +1,51 @@
+use std::net::TcpStream;
+
+fn main() {
+    // REMINDER this is dangerous, the credentials are sent over the connection in CLEARTEXT!
+    // Anyone or anything between this connection and the server can read your login creds!
+    // Please oh please do not use this where this is even a possibility.
+    match plaintext() {
+        Ok(conn) => {
+            eprintln!("Connection successful!");
+            println!("{:?}", conn);
+        }
+        Err(e) => {
+            eprintln!("Connection error!");
+            eprintln!("{:?}", e);
+        }
+    }
+}
+
+fn plaintext() -> imap::error::Result<Option<String>> {
+    let stream = TcpStream::connect("imap.example.com:143").unwrap(); // This is unsafe.
+    let mut client = imap::Client::new(stream);
+    client.read_greeting()?;
+    eprintln!("\nUNENCRYPTED connection made!!!!\n");
+    eprintln!("This is highly not recommended.\n");
+    // to do anything useful with the e-mails, we need to log in
+
+    // Makes an unencrypted login message to the IMAP server. This is risky business!
```

Comment:
The formatting here is a little odd. Let's be consistent about upper-casing in the comments (all lower-case is fine), and maybe remove the empty line?

I'd also move the risky business comment to its own line. Like so:
```suggestion
    // to do anything useful with the e-mails, we need to log in
    // keep in mind that this is over plain TCP, so may leak all your secrets!
```